Lib/pathlib.py

```diff
@@ -964,9 +1006,7 @@ def open(self, mode='r', buffering=-1, encoding=None,
         Open the file pointed by this path and return a file object, as
         the built-in open() function does.
         """
-        if "b" not in mode:
-            encoding = io.text_encoding(encoding)
-        return io.open(self, mode, buffering, encoding, errors, newline)
+        return self._unsupported("open")
```

Comment:
Minor: should these just be ``self._unsupported("...")``? (Removing the ``return``statement)

There's no functional difference, but seeing a ``return`` hints to a reader that there is a returned value, whereas these functions unconditionally raise an exception.

A

Reply:
Thanks, I've adjusted this to remove the `return`s!